examples/big_bank.py

```diff
@@ -244,117 +244,120 @@ def create_big_bank_workspace():
-    # ).Add(apiApplication)
-
-    # primaryDatabaseServer = bigBankDataCenter.add_deployment_node(
-    #     "bigbank-db01",
-    #     "The primary database server.",
-    #     "Ubuntu 16.04 LTS",
-    #     1,
-    #     DictionaryUtils.Create("Location=London"),
-    # ).add_deployment_node(
-    #     "Oracle - Primary", "The primary, live database server.", "Oracle 12c"
-    # )
-    # primaryDatabaseServer.Add(database)
-
-    # bigBankdb02 = bigBankDataCenter.add_deployment_node(
-    #     "bigbank-db02",
-    #     "The secondary database server.",
-    #     "Ubuntu 16.04 LTS",
-    #     1,
-    #     DictionaryUtils.Create("Location=Reading"),
-    # )
-    # bigBankdb02.tags.add(FailoverTag)
-    # secondaryDatabaseServer = bigBankdb02.add_deployment_node(
-    #     "Oracle - Secondary",
-    #     "A secondary, standby database server, used for failover purposes only.",
-    #     "Oracle 12c",
-    # )
-    # secondaryDatabaseServer.tags.add(FailoverTag)
-    # secondaryDatabase = secondaryDatabaseServer.Add(database)
-
-    # # model.Relationships.Where(r=>r.Destination.Equals(secondaryDatabase)).ToList().ForEach(r=>r.tags.add(FailoverTag))
+    # deployment nodes and container instances
+    developer_laptop = model.add_deployment_node(
+        environment="Development",
+        name="Developer Laptop",
+        description="A developer laptop.",
+        technology="Microsoft Windows 10 or Apple macOS",
+    )
+    apache_tomcat = developer_laptop.add_deployment_node(
+        name="Docker - Web Server",
+        description="A Docker container.",
+        technology="Docker",
+    ).add_deployment_node(
+        name="Apache Tomcat",
+        description="An open source Java EE web server.",
+        technology="Apache Tomcat 8.x",
+        instances=1,
+        properties={"Xmx": "512M", "Xms": "1024M", "Java Version": "8"},
+    )
+    apache_tomcat.add_container(web_application)
+    apache_tomcat.add_container(api_application)
+
+    developer_laptop.add_deployment_node(
+        "Docker - Database Server", "A Docker container.", "Docker"
+    ).add_deployment_node(
+        "Database Server", "A development database.", "Oracle 12c"
+    ).add_container(
+        database
+    )
+
+    developer_laptop.add_deployment_node(
+        "Web Browser", "", "Chrome, Firefox, Safari, or Edge"
+    ).add_container(single_page_application)
+
+    customer_mobile_device = model.add_deployment_node(
+        "Customer's mobile device", "", "Apple iOS or Android", environment="Live"
+    )
+    customer_mobile_device.add_container(mobile_app)
+
+    customer_computer = model.add_deployment_node(
+        "Customer's computer",
+        "",
+        "Microsoft Windows or Apple macOS",
+        environment="Live",
+    )
+    customer_computer.add_deployment_node(
+        "Web Browser", "", "Chrome, Firefox, Safari, or Edge"
+    ).add_container(single_page_application)
+
+    bigBankDataCenter = model.add_deployment_node(
```

Comment:
I prefer to not have camel case here (didn't change the variable names before since they were commented) but follow PEP8 for this and further variables names.

```suggestion
    big_bank_data_center = model.add_deployment_node(
```